alfred-telemetry-solr/alfred-telemetry-solr4/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrMetrics.java

```diff
@@ -0,0 +1,35 @@
+package eu.xenit.alfred.telemetry.solr.monitoring.binder;
+
+import eu.xenit.alfred.telemetry.solr.util.Util;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.MeterBinder;
+import org.alfresco.solr.AlfrescoCoreAdminHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.management.MBeanServer;
+
+public class SolrMetrics implements MeterBinder {
+
+    AlfrescoCoreAdminHandler coreAdminHandler;
```

Comment:
should be `private` ?

Reply:
Why?

Reply:
I don't see any package-usages ? :shrug:

Reply:
It's used by eu.xenit.alfred.telemetry.solr.monitoring.handler.MicrometerHandler, outside the package.

Reply:
Will merge now and come back to this later if needed

Reply:
If used elsewhere, please create and use an accessor. 

https://stackoverflow.com/a/1568230